build-system/tasks/check-types.js

```diff
@@ -30,7 +30,12 @@ const {extensions, maybeInitializeExtensions} = require('./extension-helpers');
 const {log} = require('../common/logging');
 const {typecheckNewServer} = require('../server/typescript-compile');
 
-const EXTERNS_GLOB = 'src/core{,/**}/*.extern.js';
+/**
+ * Files that pass type-checking but don't belong to a passing directory target.
```

Comment:
Can you add a note that this is purely a temporary measure?

Reply:
Done